frontend/src/component/Account/UserSpacesConfig.jsx

```diff
@@ -0,0 +1,208 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+import { translate } from 'react-i18next'
+
+import {
+  sortWorkspaceList,
+  ConfirmPopup,
+  TracimComponent,
+  TLM_ENTITY_TYPE as TLM_ET,
+  TLM_CORE_EVENT_TYPE as TLM_CET
+} from 'tracim_frontend_lib'
+
+import { newFlashMessage } from '../../action-creator.sync.js'
+import { deleteWorkspaceMember, getUserWorkspaceList, getWorkspaceMemberList } from '../../action-creator.async.js'
+
+import UserSpacesConfigLine from './UserSpacesConfigLine.jsx'
+
+export class UserSpacesConfig extends React.Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      workspaceList: [],
```

Comment:
Does this workspace list have a different behavior than the one in redux? If not you should use the redux one as it would save much code here (no need to handle tlms to update this list/the list of members/…).

Reply:
this is moved code that I originally made a while ago
I think there is reason to not use the redux space list, I don't remember it right now, I'm looking into it quickly

Reply:
https://github.com/tracim/tracim/issues/3735